packages/customer-account-ui-extensions/src/extension-points/index.ts

```diff
@@ -1,9 +1,11 @@
 export type {ExtensionPoints, ExtensionPoint} from './extension-points';
-export type {RenderExtension} from './render-extension';
 export type {StandardApi, Language, Localization} from './standard-api';
 export type {
-  ApiForRenderExtension,
```

Comment:
This is a breaking change, however the returns app is the only consumer and the fix is rather trivial 

```diff
diff --git a/src/type.ts b/src/type.ts
index eefe916..7362d33 100644
--- a/src/type.ts
+++ b/src/type.ts
@@ -1,6 +1,6 @@
-import type {ApiForRenderExtension} from '@shopify/customer-account-ui-extensions';
+import type {ApiForExtension} from '@shopify/customer-account-ui-extensions';
 
-export type ExtensionApi = ApiForRenderExtension<
+export type ExtensionApi = ApiForExtension<
   'CustomerAccount::FullPage::RenderWithin'
 >;
```